docs/get-started/Figma prototypes.md

```diff
@@ -15,7 +15,7 @@ However, Figma is also a helpful tool for quickly creating and collaborating on
 ## How to add the Figma library to your file
 
 1. In your Figma file, select the Figma "F" logo in the top left corner.
-2. Select **Libraries**.
+2. Select **Libraries**. Another way to reach your libraries is to select the book icon shown in the image.
```

Comment:
@Graham-Pembrey I think this also works, since the icon is in the image as well. Super :)